portfolio/src/main/webapp/script.js

```diff
@@ -50,15 +54,26 @@ function createMap() {
 
   map.controls[google.maps.ControlPosition.TOP_LEFT].push(
     document.getElementById('search-reports'));
+
+  map.controls[google.maps.ControlPosition.TOP_LEFT].push(
+    document.getElementById('search-route'));
+  
   var autocomplete = new google.maps.places.Autocomplete(
     document.getElementById('searchBox-input'));
 
+  var autocomplete2 = new google.maps.places.Autocomplete(
```

Comment:
autocomplete2 = ...getElementById('d') is unreadable. If you don't look at the HTML file, would you be able to tell what this variable has?

Rename autocomplete, autocomplete2, autocomplete3 to something descriptive (like destinationInput)